frontend/src/lang/en.ts

```diff
@@ -0,0 +1,14 @@
+export const en = {
+  translation: {
+    "Patient header": "COVID-19 Testing Portal",
+    "Register for your test": "Register for your test",
+    "Registration complete": "Registration complete",
+    terms: {
+      label: "Terms of service",
+      explanation: `This testing site uses <0>SimpleReport</0> to manage COVID-19 testing and reporting. The terms below explain SimpleReport's policies and terms of service.`,
+      intro: `As a Testing Facility (Facility) or its user (Facility User) accessing or using SimpleReport (Application) provided by the Centers for Disease Control and Prevention (CDC) and the U.S.Department of Health and Human Services (HHS), in a CDC cloud environment ("CDC Platform"), you acknowledge and agree that you are solely responsible for and shall abide by these Terms of Service, as well as any relevant sections of <0>CDC's Privacy Policies</0> (collectively, Terms).`,
+    },
+  },
+};
+
+export type LanguageConfig = typeof en;
```

Comment:
This is of my own creation; not sure if the library has a better way to enforce this. By exporting the `typeof en` as `LanguageConfig` and importing it into the Spanish translation file, we can make sure there's a corresponding Spanish translation for each English translation.

Reply:
I have a script for comparison the translation files